### PR TITLE
V2 ianm

### PR DESCRIPTION
**Pull request by purplepeopleeater**
Updates for v2 documentation

### CODE REVIEW

File: docs/Data_Portal/Users_Guide/clinical_data_analysis.md [hunk 1]
```diff
@@ -48,6 +48,33 @@ Note that the histogram plot applies to, and can be displayed for, both categori
 
 ### Survival Plot
 
+he survival analysis, which is seen in both the `Gene` and `Mutation` tabs, is used to analyze the occurrence of event data over time.  In the GDC, survival analysis is performed on the mortality of the cases. Thus, the values are retrieved from [GDC Data Dictionary](../../../Data_Dictionary) properties and a survival analysis requires the following fields:
```
Comment (clsackett) [inline docs/Data_Portal/Users_Guide/clinical_data_analysis.md:51]: Missing a "T" at the beginning ("he" --> "The")

Comment (clsackett) [inline docs/Data_Portal/Users_Guide/clinical_data_analysis.md:51]: Maybe rephrase "Thus, the values are retrieved from properties and a survival analysis requires the following fields:" to "Survival analysis requires the following fields with values retrieved from the [GDC Data Dictionary](../../../Data_Dictionary):

**Review by wwysoc2 — APPROVED** [review]
Looks good.  @purplepeopleeater can I merge this?

**Comment by purplepeopleeater** [comment]
Yes you can @wwysoc2
